javaslang-gwt/example/pom.xml

```diff
@@ -62,29 +62,15 @@
                 </configuration>
             </plugin>
             <plugin>
-                <groupId>org.codehaus.mojo</groupId>
+                <groupId>net.ltgt.gwt.maven</groupId>
                 <artifactId>gwt-maven-plugin</artifactId>
-                <version>${gwt.version}</version>
+                <version>${maven.gwt.plugin}</version>
+                <extensions>true</extensions>
                 <configuration>
-                    <logLevel>INFO</logLevel>
-                    <style>PRETTY</style>
-                    <hostedWebapp>${project.build.directory}/www</hostedWebapp>
-                    <draftCompile>false</draftCompile>
-                    <disableCastChecking>true</disableCastChecking>
-                    <disableClassMetadata>true</disableClassMetadata>
-                    <disableRunAsync>true</disableRunAsync>
-                    <extraJvmArgs>-Xms2048m</extraJvmArgs>
-                    <extraJvmArgs>-Xmx2048m</extraJvmArgs>
-                    <inplace>false</inplace>
+                    <skipModule>true</skipModule>
+                    <moduleName>JavaslangExample</moduleName>
+                    <jvmArgs>-Xmx2048m</jvmArgs>
                 </configuration>
-                <executions>
```

Comment:
Without `<executions>` and without a special `<packaging>`, the plugin won't do anything unless explicitly invoked from the command-line. Actually, I don't quite understand why this sample has `<packaging>jar</packaging>` (and includes the `maven-bundle-plugin`); I think this change is missing switching the packaging to `gwt-app`.